src/Tape.hpp

```diff
@@ -107,19 +108,20 @@ class TapeRecord
 	template<typename T>
 	T valueToYaml(T value) { return value; }
 
-	uintptr_t valueToYaml(rgl_node_t value) { return (uintptr_t)value; }
-	uintptr_t valueToYaml(rgl_node_t* value) { return (uintptr_t)*value; }
+	uintptr_t valueToYaml(rgl_node_t value) { return (uintptr_t) value; }
+	uintptr_t valueToYaml(rgl_node_t* value) { return (uintptr_t) *value; }
 
-	uintptr_t valueToYaml(rgl_entity_t value) { return (uintptr_t)value; }
-	uintptr_t valueToYaml(rgl_entity_t* value) { return (uintptr_t)*value; }
+	uintptr_t valueToYaml(rgl_entity_t value) { return (uintptr_t) value; }
+	uintptr_t valueToYaml(rgl_entity_t* value) { return (uintptr_t) *value; }
 
-	uintptr_t valueToYaml(rgl_mesh_t value) { return (uintptr_t)value; }
-	uintptr_t valueToYaml(rgl_mesh_t* value) { return (uintptr_t)*value; }
+	uintptr_t valueToYaml(rgl_mesh_t value) { return (uintptr_t) value; }
+	uintptr_t valueToYaml(rgl_mesh_t* value) { return (uintptr_t) *value; }
 
-	uintptr_t valueToYaml(rgl_scene_t value) { return (uintptr_t)value; }
-	uintptr_t valueToYaml(rgl_scene_t* value) { return (uintptr_t)*value; }
+	uintptr_t valueToYaml(rgl_scene_t value) { return (uintptr_t) value; }
+	uintptr_t valueToYaml(rgl_scene_t* value) { return (uintptr_t) *value; }
 
-	uintptr_t valueToYaml(void* value) { return (uintptr_t)value; }
+	uintptr_t valueToYaml(void* value) { return (uintptr_t) value; }
+	uintptr_t valueToYaml(const char* value) { return (uintptr_t) value;}
```

Comment:
I recall I had some problems with nullptr. I will look into this.

Reply:
> For this case (e.g. rgl_configure_logging, I'd set hardcoded path (`tape-configure-logging.txt`).

YAML file of a tape recording is easily editable so that paths can be searched&replaced. IMHO this way is better - simpler code.

Also, I've removed that `const char*` overload. Works for me without it.

Reply:
:+1: